cpp/simulations/abm.cpp

```diff
@@ -301,8 +316,10 @@ void create_assign_locations(mio::abm::World& world)
     auto testing_min_time = mio::abm::days(2);
     auto start_date       = mio::abm::TimePoint(0);
     auto end_date         = mio::abm::TimePoint(0) + mio::abm::days(60);
-    auto probability      = 1.0;
-    auto test_type        = mio::abm::AntigenTest();
+
+    auto probability = mio::UncertainValue();
+    assign_uniform_distribution(probability, 1.0, 1.0);
+    auto test_type = mio::abm::AntigenTest();
```

Comment:
This doesn't change anything in the functionality.
If you want to use an uncertainty in the probability, then change the distribution to a non-constant distribution and draw from it when creating the testing scheme. This will then result in an uncertain initial parameter for this simulation, but will be constant at runtime.

Reply:
I change it to have a probability between 0.5-1 and can be drawn at the beginning of each simulation.

Reply:
It's done!